vc_agent

```diff
@@ -17,7 +24,7 @@ export VC_AGENT_INSTALL_DIR=$VC_HOME/bin
 export VC_DRV_MANUAL_HOST_URI="$DATABASE_URL"
 export VC_HOSTNAME="${VC_HOSTNAME:-$(hostname)}"
 
-[ -f $VC_HOME/.aws/credentials ] || make_credentials
+[ -f ~/.aws/credentials ] || make_credentials
```

Comment:
Small concern here is that this is bourne shell, and I don't believe bourne shell supports `~` - though in a lot of cases the bourne shell is a bash shell, and it does non-bourne things.  A safer bet may be just using `$HOME/` instead.